pvlib/spectrum/mismatch.py

```diff
@@ -710,3 +710,86 @@ def spectral_factor_jrc(airmass, clearsky_index, module_type=None,
         + coeff[2] * (airmass - 1.5)
     )
     return mismatch
+
+
+def spectral_factor_polo(precipitable_water, airmass_absolute, aod500, aoi,
+                         altitude, module_type=None, coefficients=None,
+                         albedo=0.2):
+    """
+    Estimation of spectral mismatch for BIPV application in vertical facades.
+    Parameters
+    ----------
+    precipitable_water : numeric
+        atmospheric precipitable water. [cm]
+
+    airmass_absolute : numeric
+        absolute (pressure-adjusted) airmass. [unitless]
+    aod500 : numeric
+        atmospheric aerosol optical depth at 500 nm. [unitless]
+    aoi : numeric
+        angle of incidence. [degrees]
+    altitude: numeric
+        altitude over sea level. [m]
+    module_type : str, optional
+        One of the following PV technology strings from [1]_:
+
+        * ``'cdte'`` - anonymous CdTe module.
+        * ``'monosi'`` - anonymous sc-si module.
+        * ``'cigs'`` - anonymous copper indium gallium selenide module.
+        * ``'asi'`` - anonymous amorphous silicon module.
+    albedo
+        Ground albedo (default value if 0.2). [unitless]
+
+    coefficients : array-like, optional
+        user-defined coefficients, if not using one of the default coefficient
+        set via the ``module_type`` parameter.
+
+    Returns
+    -------
+    modifier: numeric
+        spectral mismatch factor (unitless) which is multiplied
+        with broadband irradiance reaching a module's cells to estimate
+        effective irradiance, i.e., the irradiance that is converted to
+        electrical current.
+
+    References
+    ----------
+    [1] J. Polo and C. Sanz-Saiz, 'Development of spectral mismatch models
+    for BIPV applications in building façades', Renewable Energy, vol. 245,
+    p. 122820, Jun. 2025,:doi:`10.1016/j.renene.2025.122820`
+    """
+    if module_type is None and coefficients is None:
+        raise ValueError('Must provide either `module_type` or `coefficients`')
+    if module_type is not None and coefficients is not None:
+        raise ValueError('Only one of `module_type` and `coefficients` should '
+                         'be provided')
+    am_aoi = pvlib.atmosphere.get_relative_airmass(aoi)
```

Comment:
Is it correct that `aoi = 90 - zenith`? I think so, because the surface is assumed to be vertical.  If that's the case, then I think requesting `zenith` rather than `aoi` as input would be significantly easier for a user to understand.

Reply:
I haven't read the reference so perhaps I miss something, but aoi and zenith being complementary also requires the solar azimuth to be equal to the surface azimuth, which in general won't be the case, right?